### PR TITLE
Adds Radioisotope Thermoelectric Generator - RTG

### PR DESCRIPTION
**Pull request by Tritaris**
It is a power source and heater in one. The idea being it will never be as good as a stand alone heater or power generator. Current image is a place holder til someone one can make one fitting our style.

### CODE REVIEW

**Comment by sboigelot** [comment]
Wow that's a huge picture, even for a temporary one. Care to scale it to 10% a max?

**Comment by sboigelot** [comment]
You are missing localisation for en_US

**Comment by sboigelot** [comment]
Also, where did you find the image, do we have to add a licence for it or is it yours?

**Comment by sboigelot** [comment]
Other then that the furniture works fine :+1:

**Comment by Tritaris** [comment]
The image is something I drew based on the [MMRTG](https://en.wikipedia.org/wiki/Multi-Mission_Radioisotope_Thermoelectric_Generator).

**Comment by sboigelot** [comment]
@Tritaris great then, as soon as you add a localisation in ProjectPorcupine\Assets\StreamingAssets\Localization\en_US.lang we are good to merge.

We will obviously have to discuss gameplay balance later, but that's the case for most things

**Comment by bjubes** [comment]
@Tritaris ProjectPorcupine\Assets\StreamingAssets\Localization\en_US.lang doesnt exist in this repo.

**Comment by sboigelot** [comment]
@bjubes yes I was asking it to the discord now, this changed since last time I added a text

**Comment by bjubes** [comment]
found them!
https://github.com/QuiZr/ProjectPorcupineLocalization/tree/Someone_will_come_up_with_a_proper_naming_scheme_later

**Comment by sboigelot** [comment]
Adding localisation will be discussed with the team, it is in a separate repo for now.

File: Assets/StreamingAssets/LUA/Furniture.lua [hunk 1]
```diff
@@ -659,5 +659,34 @@ function OreMine_GetSpriteName(furniture)
...
+end
+
+-- This function gets called once, when the furniture is uninstalled
+function Rtg_UninstallAction( furniture, deltaTime)
+    furniture.EventActions.Deregister("OnUpdateTemperature", "Rtg_UpdateTemperature")
+	World.Current.temperature.DeregisterSinkOrSource(furniture)
+	-- TODO: find elegant way to unregister previous register
+end
+
+function Rtg_UpdateTemperature( furniture, deltaTime)
```
Comment (Dormanil) [inline Assets/StreamingAssets/LUA/Furniture.lua]: Okay... this is literally the same code as Heater_UpdateTemperature(furniture, deltaTime). So why don't you call that one instead, either from Rtg_UpdateTemperature or directly?

**Pull request merged**